Library/Homebrew/cask/lib/hbc/cli/search.rb

```diff
@@ -1,3 +1,5 @@
+require "cmd/search"
```

Comment:
Trying to avoid adding more of these: https://github.com/Homebrew/brew/issues/4059